openstack/rds/v3/backups/requests.go

```diff
@@ -170,14 +170,37 @@ type RestoreToNewOpts struct {
 	RestorePoint     RestorePoint              `json:"restore_point" required:"true"`
 }
 
+type RestorePITROpts struct {
+	Source *Source `json:"source"`
+	Target *Target `json:"target"`
+}
+
+type Source struct {
+	InstanceId   string `json:"instance_id" required:"true"`
+	RestoreTime  int64  `json:"restore_time" required:"true"`
+	Type         string `json:"type" required:"true"`
```

Comment:
Adding the `backup_id` field will make this universal for both point-in-time and backup id restoration (in this case both fields should not have `required:"true"` tag).

Don't forget to rename a structure in this case :)